setup.py

```diff
@@ -21,6 +21,7 @@
     'torch>=1.1',
     'torchvision',
     'tqdm',
+    'h5py'
```

Comment:
Maybe this can be added as an optional requirement, as it's more like an extension to the library.

Then we can add it to `requirements-dev.txt` for the tests and raise `ImportError`s if they're needed for whatever the user is trying.

Reply:
Okay.

Reply:
If its part of the Image class, then it has to be inside setup.py as this will be used for defining the input type for the hdf5_dataset variable (having import inside image class). If we keep it as a subclass of Image, then I can only add the import inside that class.

Reply:
Is it possible to do the `import` within that class?